### DISCUSSION TITLE
Desktop App?

### DISCUSSION THREAD

**Discussion by PraveshKunwar** [monkeytypegame/monkeytype#2036] [Ideas]
Hey, I have been a user on Monkey type for a while now, and I absolutely love this app. I was wondering if there were any plans on building a desktop app? Or maybe there's already an existing one? I think it would be cool if there was a desktop app version, which could be used offline as well.

**Comment by tajhans** [comment]
This is a quick working prototype that I made that could be want you are talking about. (Its a swift playground)
[monkeytype.playgroundbook.zip](https://github.com/Miodec/monkeytype/files/7435276/monkeytype.playgroundbook.zip)

**Comment by unrealapex** [comment]
This question has been answered in #1018.

**Reply by PraveshKunwar** [reply]
thanks
